add callout for collections synching (#6594)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud-integrations/configure-auto-exposures.md

   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
4. Select the collections you want to include for auto exposures. 
   
   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>

> 

Author: ```suggestion

#### Synch collections
```

      :::info

>       dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).

Reviewer: With what you've added below, does this make more sense?
```suggestion
      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next sync (automatically once per day).
```

   
      dbt Cloud immediately starts a sync when you update the selected collections list, capturing new workbooks and removing irrelevant ones.
      :::

> 6. Click **Save**. 

Reviewer: Shouldn't this still be 5?

>       A sync will also begin immediately when the list of selected collections is modified, to capture any newly-selected workbooks and remove those which are no longer relevant.

Author: making it clear its the list in cloud
```suggestion
      dbt Cloud immediately starts a sync when you update the selected collections list, capturing new workbooks and removing irrelevant ones.
```

   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
4. Select the collections you want to include for auto exposures. 
   
   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>

> #### Sync collections

Author: ```suggestion
```

> #### Synch collections

Author: ```suggestion
#### Sync collections
```

>    #### Sync collections

Author: ```suggestion
```

>    :::info Syncing collections

Author: ```suggestion
   :::info
```

      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
   

>       To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your selections or deselections, and then click **Save**.

Author: ```suggestion
      To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your checkbox selections or deselections, and then click **Save**. You should see the updates in Explorer.
```

>       To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your checkbox selections or deselections, and then click **Save**. You should see the updates in Explorer.

Author: @joellabes @eddowh - do they have to refresh their browser in explorer or will it just show up?

>       To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your checkbox selections or deselections, and then click **Save**. You should see the updates in Explorer.

Reviewer 2: ```suggestion
      A sync will also begin immediately when the list of selected collections is modified, to capture any newly-selected workbooks and remove those which are no longer relevant.
```
